src/dataflow-types/src/proto/logging.rs

```diff
@@ -0,0 +1,240 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
```

Comment:
nit: since the original package (`crate::logging`), we should follow the convention to put the following module description of the corresponding `crate::proto::*` module:

```rust
//! Protobuf structs mirroring [`crate::logging`].
```

(for private modules, you will have to drop the auto-link brackets)

```rust
//! Protobuf structs mirroring `crate::logging`.
```